tests/test_layers.py

```diff
@@ -463,6 +463,32 @@ def test_tf_tile(self):
         expected = model.predict(data)
         self.assertTrue(run_onnx_runtime('onnx_tile', onnx_model, data, expected, self.model_files))
 
+    def test_tf_transpose(self):
+        model = Sequential()
+        model.add(Lambda(lambda x: tf.transpose(x, perm=[0, 2, 3, 1]), input_shape=[2, 3, 4]))
+        onnx_model = keras2onnx.convert_keras(model, 'test_tf_transpose')
+        data = np.random.rand(2, 2, 3, 4).astype(np.float32)
+        expected = model.predict(data)
+        self.assertTrue(run_onnx_runtime('onnx_transpose_1', onnx_model, data, expected, self.model_files))
+
+        model = Sequential()
+        model.add(Lambda(lambda x: tf.transpose(x), input_shape=[2, 3, 4]))
+        onnx_model = keras2onnx.convert_keras(model, 'test_tf_transpose')
+        data = np.random.rand(4, 2, 3, 4).astype(np.float32)
+        expected = model.predict(data)
+        self.assertTrue(run_onnx_runtime('onnx_transpose_2', onnx_model, data, expected, self.model_files))
+
+        def my_func_1(x):
+            a = tf.constant([[1, 2, 3], [4, 5, 6]], tf.float32)
+            return x + tf.transpose(a)
+
+        model = Sequential()
+        model.add(Lambda(lambda x: my_func_1(x), input_shape=[3, 2]))
```

Comment:
why not just pass in my_func_1 instead of the lambda function?

Reply:
Will do next time